status/status.go

```diff
@@ -110,9 +109,9 @@ func Code(err error) codes.Code {
 	if err == nil {
 		return codes.OK
 	}
-	if se, ok := err.(interface {
-		GRPCStatus() *Status
-	}); ok {
+
```

Comment:
Should this whole function just be `return Convert(err).Code()`?  Seems OK to lose a tiny bit of performance in an error path for maintainability purposes.  If not we probably should at least factor out these 4 lines that are common between this and `FromError`.

Reply:
The docstring for this function should also mention support for wrapped errors. So, maybe it should be something like:

```
// Code returns the Code of the error if it is a Status error or if it wraps a Status error. 
// If that is not the case, it returns codes.OK if err is nil, or codes.Unknown otherwise.
```